src/views/view-final-confirmation.njk

```diff
@@ -4,7 +4,7 @@
 {% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
 
 {% block pageTitle %}
-  View final confirmation
+  View final information
```

Comment:
why was this changed? Is this intentional?

Reply:
No, I'm not sure why this changed